public/javascripts/Admin/src/Admin.GSVCommentView.js

```diff
@@ -0,0 +1,68 @@
+function AdminGSVCommentView(admin) {
+    var self = {};
+    self.admin = admin;
+
+    var _init = function() {
+        self.panoProp = new PanoProperties();
+
+        _resetModal();
+    };
+
+    function _resetModal() {
+        var modalText =
+            '<div class="modal fade" id="labelModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">'+
+                '<div class="modal-dialog" role="document" style="width: 840px">'+
+                    '<div class="modal-content">'+
+                        '<div class="modal-header">'+
+                            '<button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>'+
+                            '<h4 class="modal-title" id="myModalLabel"></h4>'+
+                        '</div>'+
+                        '<div class="modal-body">'+
+                            '<div id="svholder" style="width: 810px; height:540px">'+
+                        '</div>'
+                        +
+                        '<div id="button-holder">' + '</div>' +
+                '</div>'+
```

Comment:
Can you make these aligned with where they normally would be like the rest of the code here?